ml-proto/spec/eval.ml

```diff
@@ -88,6 +88,9 @@ let numerics_error at = function
       error at "runtime: invalid conversion to integer"
   | exn -> raise exn
 
+let some_memory c =
+  match c.module_.memory with Some m -> m | _ -> assert false
```

Comment:
> Since impl=spec (sortof), I do wonder if we need to discriminate bugs in impl and bugs in semantics.

I think the "sortof" is key here. As much as we try to approximate
that ideal, there's still a lot of non-declarative code in the
implementation, and not everything is in the spec directory anyway. So
I think the distinction is useful.